wye/workshops/forms.py

```diff
@@ -3,7 +3,7 @@
 
 from wye.base.widgets import CalendarWidget
 
-from .models import Workshop
+from . import models
```

Comment:
@kracekumar : I see many such imports in other views and models. I think I should create a  new branch and fix all those first, so that everyone else avoids such things. What say?